scripts/pack-packages.sh

```diff
@@ -26,6 +26,11 @@ if [ -f ".releaseGroup" ]; then
   # This saves a list of the packages in the working directory in topological order to a temporary file.
   # Each package name is modified to match the packed tar files.
   flub list --no-private --releaseGroup $RELEASE_GROUP --tarball > $STAGING_PATH/pack/packagePublishOrder.txt
+
+  flub list --no-private --releaseGroup $RELEASE_GROUP --tarball --feed official > $STAGING_PATH/pack/packagePublishOrder-official.txt
+  flub list --no-private --releaseGroup $RELEASE_GROUP --tarball --feed internal > $STAGING_PATH/pack/packagePublishOrder-internal.txt
+  flub list --no-private --releaseGroup $RELEASE_GROUP --tarball --feed internal-test > $STAGING_PATH/pack/packagePublishOrder-internal-test.txt
+
```

Comment:
The previous `packagePublishOrder.txt` isn't used anymore right? Can we remove line 28 (and maybe update the comment on 26-27)?

Reply:
Left it in place intentionally for debugging purposes. I'll leave a comment.